docs/reference/nodes.md

```diff
@@ -19,7 +19,7 @@ There are five main interface elements that compose the node: the node's name, t
 To read further information about a node, `Right-Click` on the node and `Click` on the *Help* option. This will prompt a dialog with the following information:

```

Comment:
Can you check if in the Node interface section, the Node UI screenshot appears? In Github is showing the alternative text, and in the preview page is missing too.

Reply:
I was planning to address that here
https://github.com/Inbibo/shift_docs/issues/69